assets/dev/js/frontend/modules.js

```diff
@@ -19,8 +18,12 @@ elementorModules.frontend = {
 		StretchedElement,
 		SwiperBase,
 		CarouselBase,
-		NestedTabs,
 		ContactButtonsHandler,
 		FloatingBarsHandler,
 	},
 };
+
+( async () => {
+	const { default: NestedTabs } = await import( /* webpackChunkName: 'nested-tabs-module' */ 'elementor/modules/nested-tabs/assets/js/frontend/handlers/nested-tabs' );
+	elementorModules.frontend.handlers.NestedTabs = NestedTabs;
+} )();
```

Comment:
@rami-elementor

I believe that this will reduce the file size of the frontend-modules.js file, when the mega-menu is not active on the page.

Are you able to check if this reduces the file size?

---

I am still checking the async import aspect, but it seems to work correctly now.

Reply:
@rami-elementor Custom Core test passed: https://github.com/elementor/elementor-pro/actions/runs/11187111232

Reply:
@rami-elementor If this method works for the NestedTabs module, then possibly we can apply the dynamic imports for many other modules in Core and Pro as well.

Reply:
@rotemee Do you know if this is a safe method to load the Nested Tabs module, which is exted by the Mega Menu in the Pro version?

I read that there are some disadvantages of the dynamic import method: https://chatgpt.com/share/67039928-9f6c-8012-ae6f-6a841d5c47b0. 

Wdyt?